contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -36,54 +36,76 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     using SafeERC20 for IERC20;
     using ReceiptQueue for ReceiptQueue.TeleporterMessageReceiptQueue;
 
-    // SentMessageInfo includes the fee information for a given message submitted
-    // to be sent, along with the hash of the message itself.
+    /**
+     *  @notice SentMessageInfo includes the fee information for a given message submitted
```

Comment:
Super nit: There's an extra space on each line in the new comments compared to the other natspec comments in this contract.

Reply:
Oh yeah. No problem!